fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -1423,7 +1432,13 @@ public void dropPartition(Database db, Table table, DropPartitionClause clause)
                     }
                 }
             }
-            olapTable.dropPartition(db.getId(), partitionName, clause.isForceDrop());
+            Set<Long> tabletIdSet = olapTable.dropPartition(db.getId(), partitionName, clause.isForceDrop());
+            if (!tabletIdSet.isEmpty()) {
+                // for debug
+                LOG.info("delete lake tablet : {}", tabletIdSet);
+                stateMgr.getShardManager().getShardDeleter().addUnusedShardId(tabletIdSet);
+                editLog.logAddUnusedShard(tabletIdSet);
```

Comment:
tabletIdSet should be logged with DropPartitionInfo, otherwise inconsistencies may occur.